crates/api/src/proposer/api.rs

```diff
@@ -785,6 +785,46 @@ where
         Ok(get_payload_response)
     }
 
+    pub async fn get_next_active_slot(
```

Comment:
let's discuss this

Reply:
this remains right, since we are taking the slot to validate it in the commit-boost?

Reply:
I thought about adding the slot to the `/conditions` endpoint and excluding this call.